xml/System.Text/Rune.xml

```diff
@@ -1590,12 +1709,12 @@ Throws only if `input` is `null` or `index` is out of range.
         <ReturnType>System.Int32</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Returns the length in code units (<see cref="T:System.Char" />) of the UTF-16 sequence required to represent this scalar value.</summary>
-        <value>To be added.</value>
+        <summary>Gets the length in code units (<see cref="T:System.Char" />) of the UTF-16 sequence required to represent this scalar value.</summary>
+        <value>The length in code units (<see cref="T:System.Char" />) of the UTF-16 sequence required to represent this scalar value.</value>
         <remarks>
           <format type="text/markdown"><![CDATA[
 
-### Remarks
+## Remarks
 
 The return value will be 1 or 2.
```

Comment:
any explanation on why I'd get one of these numbers?

Same for the one below from 1-4.

Reply:
Here is the definition of this `Utf16SequenceLength` property:
[src/Common/src/CoreLib/System/Text/Rune.cs#L162](https://github.com/dotnet/corefx/blob/b86783ef9525f22b0576278939264897464f9bbd/src/Common/src/CoreLib/System/Text/Rune.cs#L162)

Internally, it calls `UnicodeUtility.GetUtf16SequenceLength(_value)`. That method is defined here:
[src/Common/src/CoreLib/System/Text/UnicodeUtility.cs#L45](https://github.com/dotnet/corefx/blob/b86783ef9525f22b0576278939264897464f9bbd/src/Common/src/CoreLib/System/Text/UnicodeUtility.cs#L45)

And this is its code:
```cs
/// <summary>
/// Given a Unicode scalar value, gets the number of UTF-16 code units required to represent this value.
/// </summary>
public static int GetUtf16SequenceLength(uint value)
{
    UnicodeDebug.AssertIsValidScalar(value);

    value -= 0x10000;   // if value < 0x10000, high byte = 0xFF; else high byte = 0x00
    value += (2 << 24); // if value < 0x10000, high byte = 0x01; else high byte = 0x02
    value >>= 24;       // shift high byte down
    return (int)value;  // and return it
}
```

Notice the summary is basically the same as the one we are using in this method we are documenting. So based on that, I think the remark is good enough, because the explanation is in the summary, and we are simply letting the user know that this property will only return one of two possible values.

Unless you'd prefer us to explain the details of what the method is doing, but I'm not sure the user would care.

Reply:
As discussed separately, we will open a new issue to address this later. For now, this information is enough in the remarks. @mairaw

Reply:
Issue filed: https://github.com/dotnet/dotnet-api-docs/issues/2956